api/v2/views/allocation_source.py

```diff
@@ -26,11 +26,23 @@ class AllocationSourceViewSet(MultipleFieldLookup, AuthModelViewSet):
 
     def get_queryset(self):
         """
-        Get allocation sources for user
+        Get allocation sources by user or by all users
         """
         user = self.request.user
-        sources = UserAllocationSource.objects.filter(user=user).values_list('allocation_source')
-        return AllocationSource.objects.filter(id__in=sources)
+        if user.is_admin:
```

Comment:
`is_admin` is not a property, it is a function:

https://github.com/cyverse/atmosphere/blob/zesty-zapdos/core/models/user.py#L89-L92

We should either change this to `is_admin()` or add the `@property` decorator over `is_admin`